web/src/settings_users.js

```diff
@@ -199,6 +224,22 @@ function human_info(person) {
     return info;
 }
 
+function create_role_filter_dropdown($events_container) {
+    new dropdown_widget.DropdownWidget({
+        widget_name: "select_role",
+        get_options: get_roles,
+        $events_container,
+        item_click_callback: role_selected_handler,
+        default_id: "0",
+        tippy_props: {
+            placement: "bottom-start",
+            // Without 0, 0 offset, the dropdown list starts a few pixels
+            // below the input area leaving a small gap.
+            offset: [0, 0],
```

Comment:
That's intentional.

Reply:
We should modify all the dropdowns for any options we decide to change here. I do feel like that gap is a bit much there, so maybe we can just reduce that.